apps/web/src/components/Menu/AdvancedMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import { useDynamicDrawerContext } from "@umami/components";
+import { useMemo } from "react";
+
+import { GenericMenu } from "./GenericMenu";
+import { NetworkMenu } from "./NetworkMenu";
+import { type MenuItems } from "./types";
+import { AlertCircleIcon, LockIcon, RadioIcon } from "../../assets/icons";
+
+export const AdvancedMenu = () => {
+  const { openWith } = useDynamicDrawerContext();
+
+  const menuItems = useMemo<MenuItems>(
```

Comment:
memo here is not that useful, all these values are cheap to re-render and the drawer itself is quite static anyway